spec/operators/delay-spec.ts

```diff
@@ -143,4 +146,33 @@ describe('Observable.prototype.delay', () => {
 
     expectObservable(result).toBe(expected);
   });
+
+  it('should unsubscribe scheduled action when result is unsubscribed explicitly', () => {
```

Comment:
I minor nit: the description doesn't actually describe the problem. The leak happens because the actions are not removed/unsubscribed from the subscriber's internal subscriptions after the action is executed. I think a more accurate description would be something like `should unsubscribe scheduled actions after execution`.

The actions will be unsubscribed when the subscriber unsubscribes; it's just that they will accumulate up until that happens.